cli/integration/tests/waiter/test_cli_multi_cluster.py

```diff
@@ -0,0 +1,53 @@
+import logging
+import unittest
+import uuid
+
+import pytest
+
+from tests.waiter import cli, util
+
+
+@pytest.mark.cli
+@unittest.skipUnless(util.multi_cluster_tests_enabled(), 'Requires setting WAITER_TEST_MULTI_CLUSTER')
+@pytest.mark.timeout(util.DEFAULT_TEST_TIMEOUT_SECS)
+class MultiWaiterCliTest(util.WaiterTest):
+
+    @classmethod
+    def setUpClass(cls):
+        cls.waiter_url_1 = util.retrieve_waiter_url()
+        cls.waiter_url_2 = util.retrieve_waiter_url('WAITER_URL_2', 'http://localhost:9191')
+        util.init_waiter_session(cls.waiter_url_1, cls.waiter_url_2)
+        cli.write_base_config()
+
+    def setUp(self):
+        self.waiter_url_1 = type(self).waiter_url_1
+        self.waiter_url_2 = type(self).waiter_url_2
+        self.logger = logging.getLogger(__name__)
+
+    def __two_cluster_config(self):
+        return {'clusters': [{'name': 'waiter1', 'url': self.waiter_url_1},
+                             {'name': 'waiter2', 'url': self.waiter_url_2}]}
+
+    def test_federated_show(self):
```

Comment:
Should we include tests where the token is present in only one of the two clusters?

Reply:
Done (added the test case to the same test function)